components/relevancy/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! Proposed API for the relevancy component (validation phase)
+//!
+//! The goal here is to allow us to validate that we can reliably detect user interests from
+//! history data, without spending too much time building the API out.  There's some hand-waving
+//! towards how we would use this data to rank search results, but we don't need to come to a final
+//! decision on that yet.
+
+mod db;
+mod error;
+mod interest;
+mod populate_interests;
+mod schema;
+pub mod utils;
+
+pub use db::RelevancyDb;
+pub use error::{ApiResult, Error, RelevancyApiError, Result};
+pub use interest::{Interest, InterestVector};
+
+use error_support::handle_error;
+
+pub struct RelevancyStore {
+    db: RelevancyDb,
+}
+
+impl RelevancyStore {
+    #[handle_error(Error)]
+    pub fn new(db_path: String) -> ApiResult<Self> {
+        Ok(Self {
+            db: RelevancyDb::open(db_path)?,
+        })
+    }
+
+    #[handle_error(Error)]
```

Comment:
Could you add some docstrings to those placeholder methods for future implementers?